eva-server/src/test/java/uk/ac/ebi/eva/server/ws/VariantWSServerIntegrationTest.java

```diff
@@ -131,4 +133,19 @@ public void testVepVersionAndVepCacheVersionFilter() {
         }
     }
 
+    @Test
+    public void testProteinSubstitutionScoresModel() {
+        String testVariantId = "rs370478";
+        String url = "/v1/variants/" + testVariantId + "/info?species=mmusculus_grcm38";
+        List<VariantWithSamplesAndAnnotation> variants = WSTestHelpers.testRestTemplateHelper(url, restTemplate);
+        for (VariantWithSamplesAndAnnotation variant : variants) {
+            for (ConsequenceType consequenceType : variant.getAnnotation().getConsequenceTypes()) {
+                if (consequenceType.getProteinSubstitutionScores().size() > 0) {
+                    assertNull(consequenceType.getSift());
+                    assertNull(consequenceType.getPolyphen());
```

Comment:
I am not sure this is how the model should be deserialized: if the scores are present, ideally they would populate all the fields that represent them. What I expected from the test was to check the structure of the JSON response, which is the part that is causing us problems.